TESTING.md

```diff
@@ -173,6 +173,36 @@ cargo make \
 cargo make watch-test
 ```
 
+### aws-ecs
+
+You need to [build](BUILDING.md) Bottlerocket and create an AMI before you can run a test. 
+The instance type can be configured for nvidia variants by setting `TESTSYS_INSTANCE_TYPE="p3.2xlarge"`.
```

Comment:
I would say something along these lines

```md
The default instance type to be used is <blah>, but can be controlled by setting the environment variable `TESTSYS_INSTANCE_TYPE`.
This is useful while testing NVIDIA variants, since they require instance types with support for NVIDIA GPUs.
```